### ISSUE TITLE
Sometimes camp data is blank on page 

### ISSUE THREAD

**Issue by rupalichavan9860** [the-canonizer/Canonizer-Beta--Issue-Tracking#491]
Please refer attached screenshot : 
![image](https://github.com/the-canonizer/Canonizer-Beta--Issue-Tracking/assets/40416610/7f53bb4b-430e-4b39-9107-fa13acd46b30)

And getting error on nick name link (From support tree)
![image](https://github.com/the-canonizer/Canonizer-Beta--Issue-Tracking/assets/40416610/89f48fbf-918f-48f8-a846-d203f7dc0966)

**Comment by sajid315** [comment]
I have checked both of above mentioned issues, and I found the second one only. So the second issue is coming when we are clicking from here at below URL.

https://canonizer.com/user/supports/55?topicnum=88&campnum=&namespace=1

![image](https://github.com/the-canonizer/Canonizer-Beta--Issue-Tracking/assets/111273633/3c701de2-e975-47fd-8e91-a98af197017c)

So for that @vikasranatalentelgia needs to resolve the undefined issue from the frontend and also from backend i have to verify the undefined column issue with his work.
